docs/assets/js/swagger.yml

```diff
@@ -90,7 +90,7 @@ definitions:
         type: string
         description: "Filesystem URI to store the underlying data in (i.e. 's3://my-bucket/some/path/')"
         example: "s3://example-bucket/"
-        pattern: '^(s3|mem|local|transient)://.*$'
+        pattern: '^(s3|gs|mem|local|transient)://.*$'
```

Comment:
I think this belongs to another PR?

Reply:
Yeah, not sure why this appears here as it's already in master. Will use rebase instead of merge from now on.